src/main/lrsql/util/admin.clj

```diff
@@ -46,38 +46,42 @@
   (when auth-header
     (second (re-matches #"Bearer\s+(.*)" auth-header))))
 
-(defn jwt->account-id
-  "Given the JSON Web Token `tok`, return the account ID if valid.
-   Otherwise return `:lrsql.admin/unauthorized-token-error`.
+(defn jwt->payload
+  "Given the JSON Web Token `tok`, unsign and verify the token using `secret`.
+   Return a map of `:account-id` and `:expiration` if valid, otherwise return
+   `:lrsql.admin/unauthorized-token-error`.
    `leeway` is a time amount (in seconds) provided to compensate for
    clock drift."
   [tok secret leeway]
   (if tok ; Avoid encountering a null pointer exception
     (try
-      (-> tok (bj/unsign secret {:leeway leeway}) :acc u/str->uuid)
+      (let [{:keys [acc exp]} (bj/unsign tok secret {:leeway leeway})]
+        {:account-id (u/str->uuid acc)
```

Comment:
Okay I see it now. payload is a weird word for that fn though.

Reply:
Should it be called `jwt->claim` instead?

Reply:
I will say that "payload" is the term used by the JWT website itself. And I think it's a much clearer word than "claim."

![Screenshot 2024-11-13 at 1 59 09 PM](https://github.com/user-attachments/assets/c62fd98b-ca02-457a-bb4d-14dbb39f98bf)

Reply:
it's fine. i think i just got turned around by the function changes.